forgevtt-module.js

```diff
@@ -238,13 +238,22 @@ class ForgeVTT {
                 // connection from the server side, we instead hijack the `Setup.post` on the client side so if a package is installed
                 // successfully and synchronsouly (a Bazaar install, not a protected content), we can fake a progress report
                 // of step "Package" which vends the API result.
-                if (ForgeVTT.utils.isNewerVersion(ForgeVTT.foundryVersion, "9")) {
-                    const origPost = Setup.post;
-                    Setup.post = async function (data, ...args) {
-                        const request = await origPost.call(this, data, ...args);
+
+                const preparePostOverride = (origPost) =>
+                    async function (data, ...args) {
+                        const request = await origPost.call(
+                            this,
+                            data,
+                            ...args,
+                        );
```

Comment:
I'm not super sure why, but my editor wouldn't allow me to have long lines. Outside of this function being moved and spread out over many more lines, nothing else should be modified.